release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -217,6 +217,23 @@ The Memory Manager feature is now enabled by default for all pods running on the
 
 For more information, see xref:../scalability_and_performance/using-topology-manager.adoc#topology_manager_policies_using-topology-manager[Topology Manager policies].
 
+[id="ocp-4-9-scale-latency-tools-for-latency-testing"]
+==== New tools for latency testing
+
+{product-title} {product-version} introduces two tools to measure system latency:
+
+* `hwladetect`
+* `cyclictest`
+
+This release also introduces the following environment variables:
+
+* `CYCLICTEST_MAXIMUM_LATENCY`
+* `HWLATDETECT_MAXIMUM_LATENCY`
+* `MAXIMUM_LATENCY`
+* `LATENCY_TEST_CPUS`
```

Comment:
A suggestion for each new tool and environment variable, is to maybe give a one-line description of each.
For an example, see the section on 'New descheduler profiles and customization' a little further up in the Release Notes.

Reply:
Descriptions are provided in the main section where we're linking to. Shouldn't we be short and straightforward in a release note?

Reply:
I would drop the env vars form the release note and instead add a single line description of those tools indeed.